EIPS/eip-2070.md

```diff
@@ -11,37 +11,29 @@ requires: 1679
 
 ## Abstract
 
-This meta-EIP specifies the changes included in the Ethereum hardfork named Berlin.
+This meta-EIP specified the changes included in the Ethereum hardfork named Berlin. These changes are now tracked as part of the [eth1.0-specs repositoruy](https://github.com/ethereum/eth1.0-specs/tree/master/network-upgrades). The list of EIPs included in Berlin and current implementation status is specified in [`berlin.md`](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/berlin.md).
```

Comment:
Typo "repositoruy". This typo also appears below several times.

Reply:
Thanks @poemm!

Reply:
Actually, re-reading that, I think "specified" is right: the EIP specified (in the past) the Berlin spec, it no longer does. I'll change the phrasing of the second sentence for clarity :+1: